front/app/containers/Authentication/useSteps/stepConfig/typings.ts

```diff
@@ -10,23 +10,12 @@ export type Step =
   // old sign up flow
   | 'sign-up:auth-providers'
   | 'sign-up:email-password'
-  | 'sign-up:built-in'
-  | 'sign-up:email-confirmation'
-  | 'sign-up:change-email'
-  | 'sign-up:verification'
-  | 'sign-up:custom-fields'
   | 'sign-up:invite'
 
-  // onboarding sign up flow
-  | 'sign-up:onboarding'
-
   // light flow
   | 'light-flow:email'
   | 'light-flow:email-policies'
-  | 'light-flow:google-policies'
-  | 'light-flow:facebook-policies'
-  | 'light-flow:azure-ad-policies'
-  | 'light-flow:azure-ad-b2c-policies'
```

Comment:
This gives a good overview of the main changes in the PR: instead of the `sign-up` flow having its own email confirmation, change email, verification etc steps, it now shares these from the `missing-data` flow (which is already how all the other flows worked too). There was practically no difference between these flows, so I merged them.

The other thing is that all the light flow policies now share the same step, and instead use a state variable to remember which SSO method was chosen.